README.md

```diff
@@ -1133,14 +1128,14 @@ You can run the bugfix tests from the build directory:
 
 ```bash
 $ cd <exiv2dir>/build
-$ ctest -R bugfix  
+$ ctest --test-dir build -R bugfix  
 ```
 
 If you wish to run in verbose mode:
 
 ```bash
 $ cd <exiv2dir>/build
```

Comment:
Remove `/build`

Reply:
Removed in 34fddc8f5d808893b358f829d92b70a9c282ec05